db.go

```diff
@@ -237,8 +237,18 @@ func newModel(db *DB, t reflect.Type, table Table) (*Model, error) {
 	return m, nil
 }
 
-func (m *Model) selectAll() *SelectBuilder {
-	return m.Select(m.C(m.mappedColNames...))
+// All builds a val expression to select all columns mapped by the model.
+func (m *Model) AllMapped() ValExprBuilder {
+	return m.C(m.mappedColNames...)
+}
+
+// All builds a val expression to select all columns on the models Table.
+// WARNING: This is probably not the method you want.
```

Comment:
// WARNING: This is probably not the method you want, because it
// always includes unmapped columns, even if IgnoreUnmappedCols is
// true. Old behavior kept for compatibility.